docs/writing-tests/testharness.md

```diff
@@ -198,15 +182,83 @@ Use `// META: script=link/to/resource.js` at the beginning of the resource. For
 
 can be used to include both the global and a local `utils.js` in a test.
 
-### Specifying a timeout of long in auto-generated boilerplate tests
+### Specifying a timeout of long
 
 Use `// META: timeout=long` at the beginning of the resource.
 
-### Specifying test [variants](#variants) in auto-generated boilerplate tests
+### Specifying test [variants](#variants)
```

Comment:
It's a little odd to mention variants here before they're introduced below.

Reply:
Should I make it part of the variants section somehow? It definitely seems better to have variants listed later on.